G.A.M.M.A/modpack_addons/A Few Random Fixes/gamedata/scripts/z_fluid_aim.script

```diff
@@ -0,0 +1,11 @@
+function fluid_aim.on_game_start()
+	-- this one generates the error
+	RegisterScriptCallback("actor_on_first_update", fluid_aim.ignore_fire_key_press)
+
+  -- these are fine and identical to the original
+  RegisterScriptCallback("on_key_press", fluid_aim.on_key_press)
+	RegisterScriptCallback("on_key_release", fluid_aim.on_key_release)
+	RegisterScriptCallback("actor_on_update",fluid_aim.actor_on_update)
+	RegisterScriptCallback("actor_on_weapon_jammed",fluid_aim.actor_on_weapon_jammed)
+	RegisterScriptCallback("on_option_change", fluid_aim.ignore_fire_key_press)
+end
```

Comment:
https://github.com/bellyillish/afrf/blob/main/README.md#f3

This originates here in the Fluid Aim mod (it should be "actor_on_first_update" instead).

![4](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/517d5293-73af-4ab1-bd2d-c8e20c29dde9)

Then this doesn't get called:

![5](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/a8d25e07-327a-463e-a505-5365fb78c710)

I couldn't tell just by looking at the code what this does, but it looks like some sort of keybind fix when first loading?